design/thin-edge-core.md

```diff
@@ -0,0 +1,219 @@
+# Thin-edge Design Principles
+
+__Thin-edge makes it easy the integration of cloud services, edge computing and operational technologies,
+with the foundations to develop business-specific IIoT gateways
+from a catalog of generic and use-case specific components assembled
+on top of a framework that enables connectivity and interoperability.__
+
+With IIoT, there is a fantastic opportunity for innovative use-cases and business models
+that are more reactive, flexible, and efficient.
+However, the challenges are many and each require a different expertise.
+A typical IoT application has to address:
+
+* __sophisticated business-cases__, with applications involving several vendors, actors and users
+  and combining connectivity, device management, telemetry, monitoring, analytics;
+* __very diverse integration needs__ over operating systems, protocols, data sources, processing tools,
+  actuators, cloud end-points, legacy systems;
+* __constraint environments__ with low-resources devices, high exposure to security risks,
+  prohibitive manual operations, restricted upgrades;
```

Comment:
What is a low-resource device? Is this still a device running linux? 

Maybe this is not the place for this next question but: what are the minimum specs the device should have? How much RAM for example